admin/app/templates/views/terms-of-use.html

```diff
@@ -13,46 +13,393 @@
+      more.
+    </p>
+    <h2 class="heading-medium" id="security">Security</h2>
+    <p>
+      Notify is built for the needs of government services. It has processes in
+      place to:
+    </p>
+    <ul class="list list-bullet">
+      <li>protect user data</li>
+      <li>keep systems secure</li>
+      <li>
+        manage risks around information
+      </li>
+    </ul>
+    <p>
+      System administrators of Notify have, at a minimum, a Baseline Level
+      security clearance as issued by the Australian Government Security Vetting
+      Agency.
+    </p>
+    <p>
+      <a href="mailto:notify-support@digital.gov.au">Contact us</a> to request a
+      copy of the Notify penetration test summary.
+    </p>
+    <p>
+      <strong>
+        Permissions and Two-factor authentication
+      </strong>
+    </p>
+    <p>
+      You can set different user permissions in Notify to control who in your
+      team has access to certain parts of the service.
+    </p>
+    <p>
+      Notify uses two-factor authentication. To sign in to Notify, you’ll need
+      to enter:
+    </p>
+    <ul class="list list-bullet">
+      <li>
+        your government email address and your Notify password
+      </li>
+      <li>
+        a text message code that Notify sends to your mobile phone
+      </li>
+    </ul>
+    <p><strong>Open source</strong></p>
+    <p>
+      In line with the Digital Service Standard, all code behind Notify is open
+      source and available on
+      <a href="https://github.com/govau/notify">our GitHub repository</a>.
+      Notify uses JSON Web Tokens, so API keys aren’t sent over the wire.
+    </p>
+    <p><strong>How we manage risks on Notify</strong></p>
+    <p>
+      Things we do to manage risks on Notify include:
+    </p>
+    <ul class="list list-bullet">
+      <li>regular privacy reviews</li>
+      <li>security impact assessments</li>
+    </ul>
+    <p><strong>Raising issues</strong></p>
+    <p>
+      Anyone can raise an issue for Notify. You can
+      <a href="mailto:notify-support@digital.gov.au">contact us</a> us to:
+    </p>
+    <ul class="list list-bullet">
+      <li>report a bug</li>
+      <li>request a feature</li>
+      <li>raise a security concern</li>
+    </ul>
+    <p>
+      Our service standard to responding to raised issues is 48 hours. We will
+      endeavour to address critical issues as soon possible, dependent on
+      resourcing restraints.
+    </p>
+    <h2 class="heading-medium" id="privacy-and-data-controls">
+      Privacy and data controls
+    </h2>
+    <p>
+      In Notify, your organisation is the data controller and the DTA the data
+      processor. Both parties are legally responsible for compliance with the
+      Privacy Act 1988 (Cth) and other relevant legislation.
+    </p>
+    <p>In Notify, data is encrypted:</p>
+    <ul class="list list-bullet">
+      <li>
+        when it passes through the service
```

Comment:
It's actually more than this since we encrypt communication to and also from our providers.